.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Release
+about: When ready to cut a release
+title: Release X.Y.Z
+labels: release
+assignees: ''
+
+---
+
+- [ ] Start a new release branch:
+```bash
+$ git flow release start X.Y.Z
+```
+- [ ] Rotate `CHANGELOG.md` (following [Keep a Changelog](https://keepachangelog.com/) principles)
+- [ ] Ensure outstanding changes are committed:
+```bash
+$ git add .
+$ git commit -m "X.Y.Z"
+```
+- [ ] Publish the release branch:
+```bash
+$ git flow release publish X.Y.Z
+```
+- [ ] Test the release branch on staging
+    - Wait for CI to cycle the release branch onto staging
+    - Thoroughly vet new features
+    - Commit any last-minute fixes
```

Comment:
The wording of this may unintentionally invite pushing out untested changes. We should consider something like "If testing on staging reveals bugs, create pull requests targeting the `release/X.Y.Z` branch, merge after review, and re-test after the changes are deployed to staging."

Reply:
Agreed. I added a subsection underneath this bullet.